demo/video_demo.py

```diff
@@ -33,8 +36,20 @@ def main():
         ('Please specify at least one operation (save/show the '
          'video) with the argument "--out" or "--show"')
 
+    # register all modules in mmdet into the registries
+    register_all_modules()
+
+    # build the model from a config file and a checkpoint file
     model = init_detector(args.config, args.checkpoint, device=args.device)
 
+    # build test pipeline
+    model.cfg.test_dataloader.dataset.pipeline[0].type = 'LoadImageFromNDArray'
+    test_pipeline = Compose(model.cfg.test_dataloader.dataset.pipeline)
+
+    # init visualizer
+    visualizer = VISUALIZERS.build(model.cfg.visualizer)
+    visualizer.dataset_meta = model.dataset_meta
```

Comment:
add a comment that the dataset_meta is loaded from the checkpoint and then pass to the model in `init_detector`